### ISSUE TITLE
{bezier} <-> b-spline conversion

### ISSUE THREAD

**Issue by dec1** [dhermes/bezier#214]
Hello.

If it possible to perform conversion of b-spline to corresponding set of bezier curves, and vice-versa? 

Also mentioned [here](https://math.stackexchange.com/questions/364054/is-it-possible-to-convert-a-b-spline-into-a-bezier-curve )

If not would it be possible to add this feature?

Thanks in Advance

**Comment by dhermes** [comment]
@dec1 No, this library (for now at least) has no knowledge of NURBS, b-splines, or any other extensions of Bezier curves.

I don't have any near or medium term plans on adding support for b-splines, but I'd be happy to help answer any questions and provide guidance on how you'd support b-splines **on top of** `bezier`.
